### DISCUSSION TITLE
Wrong description in READMY?

### DISCUSSION THREAD

**Discussion by andrey-val-rodin** [dotnet-bluetooth-le/dotnet-bluetooth-le#738] [Q&A] [answered]
README contains the following description of iOS permissions:
    _On iOS you must add the following keys to your Info.plist_

    <key>UIBackgroundModes</key>
    <array>
        
        <string>bluetooth-central</string>

        
        <string>bluetooth-peripheral</string>
    </array>

Maybe vice versa?
bluetooth-central for for server configurations and bluetooth-peripheral for connecting to devices?

**Comment by janusw** [comment, marked as answer]
I actually think it's correct. Usually the central is a GATT client and the peripheral is a GATT server (but it does not always need to be like this). See e.g. the discussion at https://devzone.nordicsemi.com/f/nordic-q-a/71/what-is-a-client-and-server-in-ble

**Reply by andrey-val-rodin** [reply]
Thank you!
